authzed/api/v2/bulk_service.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+package authzed.api.v2;
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v2";
+option java_package = "com.authzed.api.v2";
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "authzed/api/v2/core.proto";
+
+// BulkService exposes a number of APIs that can be used for managing
+// relationships in bulk.
+service BulkService {
+
+  // BulkLoadRelationships is a faster path to writing relationships many at
+  // a time. It is both batched and streaming. For maximum performance, the
+  // caller should attempt to write relationships in as close to relationship
+  // sort order as possible: (resource.object_type, resource.object_id,
+  // relation, subject.object.object_type, subject.object.object_id,
+  // subject.optional_relation)
+  // 
+  // EXPERIMENTAL
+  rpc BulkLoadRelationships(stream BulkLoadRelationshipsRequest)
+    returns (BulkLoadRelationshipsResponse) {
+      option (google.api.http) = {
+        post: "/v1/relationships/bulkload"
```

Comment:
v2

Reply:
@jzelinskie will this even work? how do I test it?